src/schema/users.ts

```diff
@@ -1983,6 +1984,9 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
 
       const oldStreakLength = await getRestoreStreakCache({ userId });
       if (!oldStreakLength) {
+        await ctx.con
+          .getRepository(Alerts)
+          .update({ userId }, { showRecoverStreak: false });
```

Comment:
Can we add a test for this case please

Reply:
Sure! Adding shortly, thanks for suggestion!

Reply:
And also reverse test that it does not get updated when there is a valid streak.